src/components/Main/Main.js

```diff
@@ -241,6 +234,14 @@ export class MainComponent {
     this.notFound = new NotFoundComponent(this.mainContentPoint);
 
     this.artist = new ArtistsComponent(this.mainContentPoint);
+
+    this.player = new MediaPlayerComponent(this.playerPoint, {
+      onDialogOpen: this.handleDialogOpen.bind(this),
+      onPlayerChangeState: this.handlePlayerChangeState.bind(this),
+      onLegalOptionClick: this.handleSetInfo.bind(this),
+      tableData: this.table
```

Comment:
1.Maybe you should pass list of songs instead of whole component. 
2.Will it work with other tables, like artist songs, album songs?